### ISSUE TITLE
Firefox mobile published tracks are not received

### ISSUE THREAD

**Issue by HarrisonKeeling** [livekit/client-sdk-js#884]
## Why
I am unable to receive any video track from my two android 13 devices:  one running firefox mobile `117.0` and one running `111.0`.

I am using livekit cloud and can see in the logs that the camera track publish is requested, published, then the peer subscribes to it however it is never received on the client.

> Note: If I did not include the [webrtc-adapter](https://www.npmjs.com/package/webrtc-adapter) pacakge, the track publish never occurs and gets stuck on publish requested

### Reproduction

- Using an android 13 device on firefox mobile `117.0` and one running `111.0`, visit https://meet.livekit.io
- On a second device, visit the same room
- Observe that the FF Mobile device only sends audio, but can see others

### System Info

```shell
livekit-client: 1.13.4
android 13.0
ff mobile: 117.0, 111.0
```

### Severity

blocking all usage of LiveKit

### Additional Information

Will hold off migration from our current webrtc implementation to livekit until it is resolved; a statistically significant set of our users are on this android + firefox configuration.

I'll spend some time looking into this too, debugging right now is a bit tedious as these devices are not in my physical possession

**Comment by lukasIO** [comment]
> Note: If I did not include the [webrtc-adapter](https://www.npmjs.com/package/webrtc-adapter) pacakge, the track publish never occurs and gets stuck on publish requested 

are you adding this yourself? 
the SDK already adds that package, so having two versions might cause conflicts,

Did you find any more clues while debugging this issue? 
I'll also set up a mobile device with FF mobile to specifically test for what you describe.

**Comment by HarrisonKeeling** [comment]
I tried with and without.

No clues yet but placed an order to get a couple of these phones on hand.  We also noticed this issue on a Samsung Galaxy K running android 10 on chrome 118.  Worked before when we were using webrtc with AWS Kinesis but does not work w/ livekit cloud

**Comment by lukasIO** [comment]
Was able to reproduce the issue, we'll take a look at what's going on.

**Comment by davidzhao** [comment]
Looks like this is related to a FF bug on Android, where it claims to support H.264 but doesn't encode correctly.

https://bugzilla.mozilla.org/show_bug.cgi?id=1802829

We'll have a server-side fix for this. The first of the two PRs is [here](https://github.com/livekit/livekit/pull/2183).

**Comment by davidzhao** [comment]
This has been fixed in v1.5.1 that's just released. Also fixed in Cloud
